qemu.confd

```diff
@@ -16,6 +16,9 @@
 # suitable for you, then you can omit that variable in your VM config.
 #
 
+# Name of virtual machine, by default /etc/conf.d/qemu.<VM_NAME>
+# Enable when you change variables containing it.
+#VM_NAME="${RC_SVCNAME#qemu.}"
```

Comment:
This will not work, `VM_NAME` is not declared to be overridable from the confd file.

Reply:
This does work for me, just tested this right now. By default VM_NAME variable is empty in confd file.

https://asciinema.org/a/bEWBPLCG12FiXhx5KRgJk50Lc video proof